docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -45,6 +45,8 @@ There are a few adjustments you need to do to use the CLI build system.
 - Change any absolute paths you have for `templateUrl`, `styleUrls` or lazy loaded NgModules to
 relative paths instead.
 - Remove any `module.id` that you have in `@Component` metadata. They aren't needed in the CLI.
+- Remove all default anonymous function export. If not it cause strange issue during compilation like:
```

Comment:
Grammar changes for smoother reading:

anonymous function exports.
^ export -> exports

it causes strange issues
^ cause -> causes & issue -> issues